frontend/src/app/landing/TeamProfile.jsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+import './style/landingPage.scss';
+
+type Props = {
+    name: String,
+};
+
+const TeamProfile = (props: Props): React.Node => {
+    return (
+        <div className="w-25 text-center">
+            {/* eslint-disable-next-line max-len,jsx-a11y/alt-text */}
+            <img src="https://st4.depositphotos.com/1012074/20946/v/450/depositphotos_209469984-stock-illustration-flat-isolated-vector-illustration-icon.jpg" />
+            <p>{props.name}</p>
+        </div>
```

Comment:
Use react bootstrap. Also i think that image is already downloaded in the project. Check inside `theme/static `folder. And look into `userProfileImage.jsx` file to see how to import and use it

Reply:
Also the file name should be lower camel case